Gemfile.lock

```diff
@@ -86,6 +86,7 @@ PATH
       unix-crypt
```

Comment:
Do you mind reverting the changes to `Gemfile.lock`? They're diverging rapidly with `master`. I can fix it up while landing.